apps/condo/domains/common/components/Table/Renders.tsx

```diff
@@ -201,7 +252,7 @@ export const getUnitNameRender = <T extends Record<string, unknown>>(intl, text:
 
     const unitName = text && unitNamePrefix ? `${unitNamePrefix} ${text}` : text
 
-    return getTableCellRenderer(search, false, null, null, null, extraTitle)(unitName)
+    return getTableCellRenderer({ search, extraTitle })(unitName)
```

Comment:
It is better to avoid creating unnecessary dictionaries

Reply:
In this case, it improves readability